css/index.css

```diff
@@ -0,0 +1,115 @@
+
+body {
+    background-color: #f5f5f5; 
+    color: #333;
+    font-family: 'Arial', sans-serif; 
+    margin: 0;
+    padding: 0;
+}
+
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;700&display=swap');
+body {
+    font-family: 'Roboto', sans-serif;
```

Comment:
This could be set within the first `body` declaration, even though it'll still work like this.

This overrides your declarations on line 5, though. Arial is never going to be used in this setting.

Reply:
Hi Rodrigo M,  

Thank you so much for your reply and all your recommendations. I’ve made all the necessary corrections. 

best regards